packages/minimongo/minimongo_tests.js

```diff
@@ -1570,7 +1574,7 @@ Tinytest.add("minimongo - projection_compiler", function (test) {
 
 Tinytest.add("minimongo - fetch with fields", function (test) {
   var c = new LocalCollection();
-  _.times(30, function (i) {
+  Array.from({length: 30}, function (_, i) {
```

Comment:
Underscore is accessible in this test file, so you might want to pick another variable name to represent the elements of the array.